src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -282,7 +296,17 @@ private bool IsFileSource()
             //that for file system, the "httpSource" is null.
             return _httpSource == null;
         }
- 
+
+        /// <summary>
+        /// Get the symbols package from the original package. Removes the .nupkg and adds .snupkg or .symbols.nupkg.
```

Comment:
nit: moved this function -- makes it harder to see if there is a change. and gives us more to review.